bicep/aks-cluster.bicep

```diff
@@ -0,0 +1,75 @@
+param dnsPrefix string
+param linuxAdminUsername string
+@secure()
+param sshRSAPublicKey string
+param subnetId string
+param location string = resourceGroup().location
+
+resource aks 'Microsoft.ContainerService/managedClusters@2023-01-01' = {
+  name: 'pocimpinfak1401'
+  location: location
+  identity: {
+    type: 'SystemAssigned'
```

Comment:
We'll want to use a user-assigned identity so that we can reliably configure permissions for the Azure CRDs (Azure Service Operator)